packages/teams-js/test/internal/communication.spec.ts

```diff
@@ -1,6 +1,6 @@
 import * as communication from '../../src/internal/communication';
-import * as handlers from '../../src/internal/handlers';
 import { GlobalVars } from '../../src/internal/globalVars';
```

Comment:
Again, automatically made by prettier. They aren't technically related to my change but since they are innocuous I left them in (and they would keep coming back in future changes until they were made, anyway)